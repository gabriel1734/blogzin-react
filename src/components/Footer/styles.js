import styled, { css } from 'styled-components';

export const Container = styled.footer`
  ${({ theme }) => css`
    text-align  : center;
  span{
    color: ${theme.colors.secondaryColor};
  }
  a {
      color: ${theme.colors.primaryColor};
      text-decoration: none;
      font-size: ${theme.fonts.sizes.small};
    }
  `}
`;
