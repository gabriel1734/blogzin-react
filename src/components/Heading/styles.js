import styled, { css } from 'styled-components';

export const Title = styled.h1`
  ${({ theme }) => css`
    background-color: ${theme.colors.mainBg};
  `}`;
