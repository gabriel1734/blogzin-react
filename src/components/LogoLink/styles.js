import styled, { css } from 'styled-components';

export const Container = styled.a`
  ${({ theme }) => css`
    display: flex;
    align-items: center;
    color: ${theme.colors.primaryColor};
    text-decoration: none;
    

    > img {
      max-height: 3rem;
    }
  `}
`;
