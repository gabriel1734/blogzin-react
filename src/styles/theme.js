export const theme = {
  colors: {
    mainBg: 'pink',
    secondaryBg: 'blue',
  },
  fonts: {},
  spacings: {},
};
