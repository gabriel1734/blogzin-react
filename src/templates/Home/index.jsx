import { useEffect, useState } from 'react';
import { Base } from '../Base';
import { mapData } from '../../api/map-data';
import { PageNotFound } from '../PageNotFound';
import { Loading } from '../../components/Loading';
import { GridTwoColumn } from '../../components/GridTwoColumn';
import { GridContent } from '../../components/GridContent';
import { GridText } from '../../components/GridText';
import { GridImage } from '../../components/GridImage';

function Home() {
  const [data, setData] = useState([]);
  useEffect(() => {
    (async () => {
      try {
        const data = await fetch(
          'http://localhost:1337/api/pages?populate=deep&pagination[pageSize]=1&sort[0]=id:desc',
        );
        const json = await data.json();
        const pageData = mapData([json.data[0].attributes]);
        setData(pageData[0]);
      } catch (e) {
        setData(undefined);
      }
    })();
  }, []);

  if (data === undefined) {
    return <PageNotFound />;
  }

  if (data && !data.slug) {
    return <Loading />;
  }

  const { menu, sections, footerHtml, slug } = data;
  const { links, text, link, srcImg } = menu;
  console.log(menu);
  return (
    <Base
      links={links}
      footerHtml={footerHtml}
      LogoData={{ text, link, srcImg }}
    >
      {sections.map((section, index) => {
        const { component } = section;
        const key = `${slug}-${index}`;
        if (component === 'section.section-two-columns') {
          return <GridTwoColumn key={key} {...section} />;
        }
        if (component === 'section.section-content') {
          return <GridContent key={key} {...section} />;
        }
        if (component === 'section.section-grid') {
          return <GridText key={key} {...section} />;
        }
        if (component === 'section.section-grid-image') {
          return <GridImage key={key} {...section} />;
        }
      })}
    </Base>
  );
}

export default Home;
