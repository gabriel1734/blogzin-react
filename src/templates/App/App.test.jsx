import Home from '.';
import { renderTheme } from '../../styles/render-theme';

describe('Home', () => {
  it('should render correctly', () => {
    renderTheme(<Home />);
  });
});
