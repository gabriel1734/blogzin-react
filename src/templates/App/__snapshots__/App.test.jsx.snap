// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly 1`] = `
.c11 {
  padding-top: 2rem;
}

.c4 {
  max-width: 120rem;
  margin: 0 auto;
  padding: 3.2rem;
}

.c7 {
  color: #0A1128;
  font-size: 2.4rem;
  text-transform: uppercase;
}

.c14 {
  color: #fff;
  font-size: 6.2rem;
  text-transform: uppercase;
}

.c19 {
  color: #fff;
  font-size: 3.2rem;
  text-transform: none;
}

.c21 {
  color: #0A1128;
  font-size: 6.2rem;
  text-transform: uppercase;
}

.c22 {
  color: #0A1128;
  font-size: 3.2rem;
  text-transform: none;
}

.c2 {
  position: fixed;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  z-index: 5;
  top: 0;
  left: 0;
  right: 0;
  padding-top: 3.5rem;
  border-bottom: #DDDDDD;
  background-color: #fff;
  -webkit-transition: all .3s ease-in-out;
  transition: all .3s ease-in-out;
  height: 10vh;
}

.c2 > .c3 {
  padding-top: 0;
  padding-bottom: 0;
}

.c2 .c6 {
  margin-top: 0;
  margin-bottom: 0;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 4.8rem 0;
}

.c0 {
  display: none;
  z-index: 6;
  position: fixed;
  top: 2rem;
  right: 2rem;
  width: 4rem;
  height: 4rem;
  background-color: #0A1128;
  color: #fff;
  border: none;
  pointer-events: all;
}

.c0 > svg {
  width: 2.5rem;
  height: 2.5rem;
}

.c8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  color: #0A1128;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c8 > img {
  max-height: 3rem;
}

.c9 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: row wrap;
  -ms-flex-flow: row wrap;
  flex-flow: row wrap;
}

.c10 {
  display: block;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-size: 2.4rem;
  padding: 1.6rem;
  color: #0A1128;
  position: relative;
}

.c10::after {
  content: '';
  position: absolute;
  bottom: 0.76rem;
  left: 50%;
  width: 0%;
  height: 0.2rem;
  background-color: #dc143c;
  -webkit-transition: all 0.5s ease-in-out;
  transition: all 0.5s ease-in-out;
}

.c10:hover::after {
  left: 25%;
  width: 50%;
}

.c1 {
  display: inline-block;
  vertical-align: middle;
  overflow: hidden;
}

.c16 {
  font-size: 2.4rem;
}

.c23 {
  text-align: center;
  border-top: 0.2rem solid #DDDDDD;
}

.c23 span {
  color: #dc143c;
}

.c23 a {
  color: #0A1128;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c23 .c15 {
  font-size: 1.6rem;
}

.c23 .c3 {
  padding-top: 0;
  padding-bottom: 0;
}

.c24 {
  position: fixed;
  bottom: 2rem;
  right: 2rem;
  z-index: 6;
  background-color: #0A1128;
  color: #fff;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 4rem;
  height: 4rem;
  opacity: 0.8;
}

.c13 .c15 {
  margin-bottom: 6.2rem;
}

.c17 {
  counter-reset: grid-counter;
  display: grid;
  grid-template-columns: repeat(auto-fill,minmax(280px,1fr));
  gap: 3.2rem;
}

.c18 .c6 {
  position: relative;
  left: 5rem;
}

.c18 .c6::before {
  counter-increment: grid-counter;
  content: counter(grid-counter);
  position: absolute;
  font-size: 7rem;
  top: -3rem;
  left: -5rem;
  -webkit-transform: rotate(5deg);
  -ms-transform: rotate(5deg);
  transform: rotate(5deg);
}

.c12 {
  background-color: #fff;
  color: #0A1128;
  background-color: #0A1128;
  color: #fff;
  min-height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c20 {
  background-color: #fff;
  color: #0A1128;
  min-height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

@media (max-width:768px) {
  .c14 {
    font-size: 4rem;
    background-color: 'red';
  }
}

@media (max-width:768px) {
  .c21 {
    font-size: 4rem;
    background-color: 'red';
  }
}

@media (max-width:1180px) {
  .c2 {
    height: 18vh;
    padding-top: 0;
  }
}

@media (max-width:768px) {
  .c2 {
    height: 100vh;
    visibility: hidden;
    opacity: 0;
  }

  .c2 > .c3 {
    display: grid;
    grid-template-columns: 1fr;
    grid-template-rows: 1fr;
    height: 100vh;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    overflow-y: auto;
  }

  .c2 .c6 {
    padding-bottom: 3.2rem;
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-box-pack: center;
    -webkit-justify-content: center;
    -ms-flex-pack: center;
    justify-content: center;
  }
}

@media (max-width:768px) {
  .c5 {
    display: block;
    text-align: center;
  }
}

@media (max-width:768px) {
  .c0 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    -webkit-box-pack: center;
    -webkit-justify-content: center;
    -ms-flex-pack: center;
    justify-content: center;
    cursor: pointer;
  }
}

@media (max-width:768px) {
  .c9 {
    -webkit-flex-flow: column wrap;
    -ms-flex-flow: column wrap;
    flex-flow: column wrap;
    -webkit-align-content: center;
    -ms-flex-line-pack: center;
    align-content: center;
  }
}

@media (max-width:768px) {
  .c17 {
    grid-template-columns: 1fr;
  }
}

<div>
  <button
    class="c0"
  >
    <svg
      aria-hidden="true"
      aria-label="Open menu"
      class="c1"
      fill="currentColor"
      focusable="false"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 0h24v24H0V0z"
        fill="none"
      />
      <path
        d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
      />
    </svg>
  </button>
  <div
    class="c2"
  >
    <div
      class="c3 c4"
    >
      <div
        class="c5"
      >
        <h1
          class="c6 c7"
        >
          <a
            class="c8"
            href="#"
            target="_self"
          >
            <span>
              Logo
            </span>
          </a>
        </h1>
        <nav
          aria-label="Main menu"
          class="c9"
        >
          <a
            class="c10"
            href="#target1"
            target="_self"
          >
            Link 1
          </a>
          <a
            class="c10"
            href="#target2"
            target="_self"
          >
            Link 2
          </a>
          <a
            class="c10"
            href="#target3"
            target="_self"
          >
            Link 3
          </a>
          <a
            class="c10"
            href="#target4"
            target="_self"
          >
            Link 4
          </a>
          <a
            class="c10"
            href="#target5"
            target="_self"
          >
            Link 5
          </a>
          <a
            class="c10"
            href="#target6"
            target="_self"
          >
            Link 6
          </a>
          <a
            class="c10"
            href="#target7"
            target="_self"
          >
            Link 7
          </a>
          <a
            class="c10"
            href="#target8"
            target="_self"
          >
            Link 8
          </a>
          <a
            class="c10"
            href="#target9"
            target="_self"
          >
            Link 9
          </a>
          <a
            class="c10"
            href="#target10"
            target="_self"
          >
            Link 10
          </a>
        </nav>
      </div>
    </div>
  </div>
  <div
    class="c11"
  >
    <div
      class="c12"
    >
      <div
        class="c3 c4"
      >
        <div
          class="c13"
        >
          <h2
            class="c6 c14"
          >
            My grid
          </h2>
          <div
            class="c15 c16"
          >
            Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde
          </div>
          <div
            class="c17"
          >
            <div
              class="c18"
            >
              <h1
                class="c6 c19"
              >
                Teste 1
              </h1>
              <div
                class="c15 c16"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="c18"
            >
              <h1
                class="c6 c19"
              >
                Teste 2
              </h1>
              <div
                class="c15 c16"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="c18"
            >
              <h1
                class="c6 c19"
              >
                Teste 3
              </h1>
              <div
                class="c15 c16"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c20"
    >
      <div
        class="c3 c4"
      >
        <div
          class="c13"
        >
          <h2
            class="c6 c21"
          >
            My grid
          </h2>
          <div
            class="c15 c16"
          >
            Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde
          </div>
          <div
            class="c17"
          >
            <div
              class="c18"
            >
              <h1
                class="c6 c22"
              >
                Teste 1
              </h1>
              <div
                class="c15 c16"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="c18"
            >
              <h1
                class="c6 c22"
              >
                Teste 2
              </h1>
              <div
                class="c15 c16"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="c18"
            >
              <h1
                class="c6 c22"
              >
                Teste 3
              </h1>
              <div
                class="c15 c16"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c12"
    >
      <div
        class="c3 c4"
      >
        <div
          class="c13"
        >
          <h2
            class="c6 c14"
          >
            My grid
          </h2>
          <div
            class="c15 c16"
          >
            Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde
          </div>
          <div
            class="c17"
          >
            <div
              class="c18"
            >
              <h1
                class="c6 c19"
              >
                Teste 1
              </h1>
              <div
                class="c15 c16"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="c18"
            >
              <h1
                class="c6 c19"
              >
                Teste 2
              </h1>
              <div
                class="c15 c16"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="c18"
            >
              <h1
                class="c6 c19"
              >
                Teste 3
              </h1>
              <div
                class="c15 c16"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c20"
    >
      <div
        class="c3 c4"
      >
        <div
          class="c13"
        >
          <h2
            class="c6 c21"
          >
            My grid
          </h2>
          <div
            class="c15 c16"
          >
            Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde
          </div>
          <div
            class="c17"
          >
            <div
              class="c18"
            >
              <h1
                class="c6 c22"
              >
                Teste 1
              </h1>
              <div
                class="c15 c16"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="c18"
            >
              <h1
                class="c6 c22"
              >
                Teste 2
              </h1>
              <div
                class="c15 c16"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="c18"
            >
              <h1
                class="c6 c22"
              >
                Teste 3
              </h1>
              <div
                class="c15 c16"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c12"
    >
      <div
        class="c3 c4"
      >
        <div
          class="c13"
        >
          <h2
            class="c6 c14"
          >
            My grid
          </h2>
          <div
            class="c15 c16"
          >
            Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde
          </div>
          <div
            class="c17"
          >
            <div
              class="c18"
            >
              <h1
                class="c6 c19"
              >
                Teste 1
              </h1>
              <div
                class="c15 c16"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="c18"
            >
              <h1
                class="c6 c19"
              >
                Teste 2
              </h1>
              <div
                class="c15 c16"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="c18"
            >
              <h1
                class="c6 c19"
              >
                Teste 3
              </h1>
              <div
                class="c15 c16"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c20"
    >
      <div
        class="c3 c4"
      >
        <div
          class="c13"
        >
          <h2
            class="c6 c21"
          >
            My grid
          </h2>
          <div
            class="c15 c16"
          >
            Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde
          </div>
          <div
            class="c17"
          >
            <div
              class="c18"
            >
              <h1
                class="c6 c22"
              >
                Teste 1
              </h1>
              <div
                class="c15 c16"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="c18"
            >
              <h1
                class="c6 c22"
              >
                Teste 2
              </h1>
              <div
                class="c15 c16"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="c18"
            >
              <h1
                class="c6 c22"
              >
                Teste 3
              </h1>
              <div
                class="c15 c16"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <footer
      class="c23"
    >
      <div
        class="c3 c4"
      >
        <div
          class="c15 c16"
        >
          <p>
            Teste de footer
          </p>
        </div>
      </div>
    </footer>
  </div>
  <a
    aria-label="Go to top"
    class="c24"
    href="#"
    title="Go to top"
  >
    <svg
      aria-hidden="true"
      class="c1"
      fill="currentColor"
      focusable="false"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 0h24v24H0V0z"
        fill="none"
      />
      <path
        d="M7.41 15.41L12 10.83l4.59 4.58L18 14l-6-6-6 6 1.41 1.41z"
      />
    </svg>
  </a>
</div>
`;
